.github/workflows/run_tests/action.yml

```diff
@@ -3,21 +3,11 @@ runs:
   steps:
     - name: Run tests
       run: |
-          cd ${GITHUB_WORKSPACE}/build
-          ./tests/test_SVD
-          ./tests/test_Vector
-          ./tests/test_Matrix
-          ./tests/test_DEIM
-          ./tests/test_GNAT
-          ./tests/test_QDEIM
-          ./tests/test_S_OPT
-          mpirun -n 4 --oversubscribe tests/test_S_OPT
-          ./tests/test_IncrementalSVD
-          ./tests/test_RandomizedSVD
-          mpirun -n 3 --oversubscribe tests/test_RandomizedSVD
-          ./tests/test_DMD
-          mpirun -n 3 --oversubscribe tests/test_DMD
-          ./tests/test_GreedyCustomSampler
-          mpirun -n 3 --oversubscribe tests/test_GreedyCustomSampler
+          cd ${GITHUB_WORKSPACE}/build/examples/prom
+          ./poisson_global_rom -offline -f 1.0 -id 0
```

Comment:
This line is executing one of the examples. It doesn't make it to here because it crashes a bit earlier in the build process.